packages/react-ui-components/src/SelectBox_Header/style.css

```diff
@@ -3,11 +3,35 @@
     display: flex;
 }
 
+.selectBoxHeader:after {
+    display: block;
+    content: '';
+    position: absolute;
+    width: 1px;
+    height: 24px;
+    top: 8px;
```

Comment:
Could use constants here as well

Reply:
I should introduce a constant for width/height/top even if they are not used anywhere else?

Reply:
No, definitely not, but a lot of numbers could be calc'ed from base constants.